scoop-basics/src/main/java/com/example/scoop/basics/AppModule.java

```diff
@@ -25,8 +26,20 @@ public AppModule(App app) {
 
     @Singleton
     @Provides
-    AppRouter provideAppRouter() {
-        return new AppRouter(new DaggerScreenScooper());
+    DaggerScreenScooper provideDaggerScreenScooper() {
+        return new DaggerScreenScooper();
+    }
+
+    @Singleton
+    @Provides
+    AppRouter provideAppRouter(DaggerScreenScooper daggerScreenScooper) {
+        return new AppRouter(daggerScreenScooper, false);
```

Comment:
@itspbj may be another contstructor with default to "false" will make sense. We don't want people overuse this flag =))